addons/position_manager/position_manager.lua

```diff
@@ -67,31 +73,48 @@ function handle_commands(cmd, pos_x, pos_y, name)
             name = 'all'
         end
 
-        settings.x = tonumber(pos_x)
-        settings.y = tonumber(pos_y)
-        if settings.x and settings.y then
-            config.save(settings, name)
-        else
-            windower.add_to_chat(207, 'plugin_manager: ERROR - invalid position provided.')
-            windower.send_command('pm help')
-            return
-        end
-
-        -- TODO: possibly add IPC
-        if windower.ffxi.get_info().logged_in then
-            player_name = windower.ffxi.get_player().name
-            if name:lower() == player_name:lower() then
-                move(settings)
+        if arg[1] == 'delay' then
+            settings.delay = tonumber(arg[2])
+            if settings.delay > 0 then
+                config.save(settings, name)
+                windower.add_to_chat(207, 'position_manager: Delay set to %s for %s.':format(settings.delay, name))
+            else
+                windower.add_to_chat(207, 'position_manager: ERROR - invalid delay provided.')
+                windower.send_command('pm help')
+                return
+            end
+        elseif tonumber(arg[1]) and tonumber(arg[2]) then
+            settings.x = tonumber(arg[1])
+            settings.y = tonumber(arg[2])
+        
+            if settings.x and settings.y then
+                config.save(settings, name)
+                
+                if windower.ffxi.get_info().logged_in then
+                    player_name = windower.ffxi.get_player().name
+                    if name:lower() == player_name:lower() then
+                        move(settings)
+                    end
+                end                
+                windower.add_to_chat(207, 'position_manager: Position set to %s,%s for %s.':format(settings.x, settings.y, name))
+            else
+                windower.add_to_chat(207, 'position_manager: ERROR - invalid position provided.')
+                windower.send_command('pm help')
+                return
             end
         end
-    elseif cmd == 'help' then
-        windower.add_to_chat(207, 'position_manager: Usage: //pm set <x> <y> [name]')
-        windower.add_to_chat(207, 'position_manager: See the readme for details.')
-    else
+        -- TODO: possibly add IPC
+        return
+    elseif cmd ~= 'help' then
         windower.add_to_chat(207, 'position_manager: %s command not found.':format(cmd))
-        windower.send_command('pm help')
-    end
+    end        
+
+    windower.add_to_chat(207, 'position_manager: Commands:')
+    windower.add_to_chat(207, '  //pm set <x> <y> [name]')
+    windower.add_to_chat(207, '  //pm set delay <seconds> [name]')
+    windower.add_to_chat(207, 'position_manager: See the readme for details.')
```

Comment:
Not sure why you moved that away from the `help` command. This way it will also print after setting a new delay or position. Was that intended? If so, you have to remember to put `return` at the end of every successful command.

If you want it to print when passing in invalid info I would suggest making a `syntax` function or similar and call that in case of an invalid input (and possibly within the `help` command as well).

Reply:
> Not sure why you moved that away from the `help` command. This way it will also print after setting a new delay or position. Was that intended? If so, you have to remember to put `return` at the end of every successful command.

It's this way because I was writing with IPC in mind - look at lines 106 and 107, there's a `return` that takes care of those two cases, the line right above is supposed to send an IPC message to update the affected character (the function at 93-98 would then go in the ipc message event, and take care of reloading settings and updating the position).

> If you want it to print when passing in invalid info I would suggest making a `syntax` function or similar and call that in case of an invalid input (and possibly within the `help` command as well).

With the above in mind, a show_help() function won't simplify the code because the if tree would be unchanged, but it'd let me get rid of the `windower.send_command('pm help')` calls, so I'll do it.